app/models/service.js

```diff
@@ -23,6 +23,24 @@ function ServiceViewModel(data, runningTasks, nodes) {
   if (data.Spec.TaskTemplate.ContainerSpec.Env) {
     this.Env = data.Spec.TaskTemplate.ContainerSpec.Env;
   }
+  // Find first manager IP
+  if (nodes) {
```

Comment:
I don't like this logic being placed inside the model. Model should just assign fields from the Docker API to the model.

This part should be done in the Controller or Service. Maybe using a ServiceHelper.